nautobot/extras/tests/test_jobs.py

```diff
@@ -77,6 +77,18 @@ def test_job_pass(self):
         job_result = create_job_result_and_run_job(module, name)
         self.assertEqual(job_result.status, JobResultStatusChoices.STATUS_SUCCESS)
 
+    def test_job_result_manager_censor_sensitive_variables(self):
+        """
+        Job test with JobResult Censored Sensitive Variables.
+        """
+        module = "test_has_sensitive_variables"
+        name = "TestHasSensitiveVariables"
+        # This function create_job_result_and_run_job and the subsequent functions' arguments are very messy
+        job_result = create_job_result_and_run_job(module, name, "local", 1, 2, "3", kwarg_1=1, kwarg_2="2")
```

Comment:
Your run method signature for `TestHasSensitiveVariables` needs to match the args/kwargs here

Reply:
Like this correct? https://github.com/nautobot/nautobot/pull/3800/commits/9ed4cd879272f922f353a6e284ab68bd9bbd6305